designs-for-feedback/trigger-workflow-in-pipeline.md

```diff
@@ -0,0 +1,38 @@
+# Request for Comment: Trigger an individual workflow
+
+## Problem statement
+Users want to be able to trigger a pipeline, selecting a specific workflow in the configuration to run. They also need a way for such manually triggered workflows to not always run when the pipeline is triggered without specifying a particular workflow.
+
+## Proposed solution
+1. We will add a parameter to the triggering of pipelines (`POST /project/:project_slug/pipeline`) to specify a particular workflow by name.
+2. We will also add a new way to specify in config to not run a particular workflow unless it is explicitly triggered.
+
+## Design options for the new triggering parameter.
+In v1.1 we combined the custom parameters passed into a build with our built-in parameters, passing them all when triggering a job as a single name/value collection. In v2 we plan to separate out your custom parameters from built-in flags and settings.
+
+When you want to trigger an individual workflow in a configuration file we can either:
```

Comment:
Should we consider a syntax that permits a user to run more than one workflow out of all of the workflows configured?

Reply:
I should have read further on.